app/views/dashboard/_application_templates.haml

```diff
@@ -1,26 +1,28 @@
-%ul
-  - @departments.each do |department|
-    %li
-      %h3= department.name
-    %ul
-      - department.positions.each do |position|
-        %li
-          = position.name
-          %ul
-            - if @templates.key? position
-              - template = @templates[position].first
-              %li= link_to 'View application',
-                  application_template_path(template)
-              %li
-                - if template.draft_belonging_to? @current_user
-                  - draft = template.draft_belonging_to @current_user
-                  = link_to 'Resume editing saved draft', edit_draft_path(draft)
-                  %br
-                  = button_to 'Discard saved draft',
-                    draft_path(draft), method: :delete
-                - else
-                  = link_to 'Edit application',
-                    new_draft_path(application_template_id: template.id)
-            - else # position has no template... yet
-              %li= link_to 'Create application',
-                    new_application_template_path(position_id: position.id)
+.dashboard_partial
```

Comment:
I wish Github were better at showing you "this just got indented inside a new thing".